RELEASING.md

```diff
@@ -1,13 +1,26 @@
-1. Create a release branch- `release/x.y.z`
-1. Update versions in `VERSIONS.md`
-1. Update to the latest purchases-hybrid-common version in `plugin.xml`
-1. Update version in `package.json`, `plugin.xml`, `PurchasesPlugin.java`, `PurchasesPlugin.swift`, and `scripts/docs/index.html`.
-1. Run `npm run build`
-1. Add an entry to CHANGELOG.md
-1. Create a PR with the changes - "Preparing for version x.y.z" and include the new contents of `CHANGELOG.md` as the description.
-1. Once approved, merge.
-1. From the updated `main` branch, create a tag `git tag x.y.z`
-1. `git push origin <tag name>`
-1. Create a new release in github and upload the output from the build. If this is a beta/release candidate, make sure to check `This is a pre-release`
-1. `npm publish`
-1. Update the version in `package.json` and `plugin.xml` to x.y.z-SNAPSHOT, with x.y.z being the next version
+Automatic Releasing
+=========
+1. Create a `fastlane/.env` file with your GitHub API token (see `fastlane/.env.SAMPLE`). This will be used to create the PR, so you should use your own token so the PR gets assigned to you.
+1. Run `bundle exec fastlane bump`
+ 1. Confirm base branch is correct
+ 2. Input new version number
+ 3. Update CHANGELOG.latest.md to include the latest changes. Call out API changes (if any). You can use the existing CHANGELOG.md as a base for formatting. To compile the changelog, you can compare the changes between the base branch for the release (usually main) against the latest release, by checking https://github.com/revenuecat/cordova-plugin-purchases/compare/<latest_release>...<base_branch>. For example, https://github.com/revenuecat/cordova-plugin-purchases/compare/2.4.0...main.
+ 4. A new branch and PR will automatically be created
+1. `bundle exec fastlane update_hybrid_common version:x.x.x` to set the purchases-hybrid-common version number if needed.
```

Comment:
I feel like this should be done on each release. Totally automatic.

Reply:
I completely agree. I believe we plan to have automatic bumps for PHC every week, but that's a separate task. For now, I left it as a separate process and left it in the instructions as a reminder in case we want to bump it to something else

Reply:
Makes sense.